integrationExamples/gpt/digitrust_min_DTinit.html

```diff
@@ -0,0 +1,281 @@
+<html>
```

Comment:
Can you add a unit test for your change to `test/spec/modules/digitrustIdSystem_spec.js`, as well?

Reply:
I will review and see if it is practical to do so, and go ahead if there are no ill effects.